modules/Bio/EnsEMBL/Variation/Utils/DbNSFPProteinFunctionAnnotation.pm

```diff
@@ -0,0 +1,209 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+
+
+=head1 CONTACT
+
+ Please email comments or questions to the public Ensembl
+ developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+ Questions may also be sent to the Ensembl help desk at
+ <http://www.ensembl.org/Help/Contact>.
+
+=head1 DESCRIPTION
+
+Module is used in protein function prediction pipeline for
+annotating all possible amino acid substitutions in a translation
+with dbNSFP (revel, meta_lr and mutation_assessor) scores and predictions.
+
+=cut
+
+
+use strict;
+use warnings;
+
+package Bio::EnsEMBL::Variation::Utils::DbNSFPProteinFunctionAnnotation;
+
+use Bio::EnsEMBL::Variation::Utils::BaseProteinFunctionAnnotation;
+our @ISA = ('Bio::EnsEMBL::Variation::Utils::BaseProteinFunctionAnnotation');
+
+my $REVEL_CUTOFF = 0.5;
+
+=head2 new
+  Example    :
```

Comment:
perldoc DbNSFPProteinFunctionAnnotation.pm does not display clearly. Adding blank line after =head2 helps.
Similarly for get_dbNSFP_row  description